docker/README.md

```diff
@@ -63,7 +71,7 @@ Unlike the `circ-scripts` image, which runs constantly and executes every possib
 
 Because containers based on `circ-exec` are built, run their job, and are destroyed, it's important to configure an external log aggregator to find &#42;.log files in `/var/log/simplified/${SIMPLIFIED_SCRIPT_NAME}.log`.
 
-```
+```sh
 # See the section "Environment Variables" below for more information
 # about the values listed here and their alternatives.
 $ docker run --name refresh-materialized-views -it \
```

Comment:
We don't have this script anymore, so while you're changing this file, change the name of the script to `search_index_refresh`.